RELEASE_NOTES.md

```diff
@@ -1,5 +1,18 @@
 # Release Notes
 
+## 5.7.1
+### Security
+- This release addresses [CVE-2025-52881](https://github.com/advisories/GHSA-cgrx-mc8f-2prm), where the `podman build` command could, when using the non-standard `chroot` isolation, allow `RUN` instructions to escape the build container.
```

Comment:
we mention the CVE under 5.7.0 already so this should likely mention what changed if we mention it again, I think we picked up a bunch of regression fixes?